src/test/java/com/puppycrawl/tools/checkstyle/utils/TokenUtilsTest.java

```diff
@@ -63,7 +67,7 @@ public void testGetIntFromInaccessibleField() throws NoSuchFieldException {
 
     @Test
     public void testTokenValueIncorrect() {
-        final Integer id = Integer.MAX_VALUE - 1;
+        final Integer id = 184;
```

Comment:
184 = largest token + 1
What was wrong with old value? I don't see right now how this survived at https://nimfadora.github.io/com.puppycrawl.tools.checkstyle.utils/TokenUtils.java.html#org.pitest.mutationtest.report.html.SourceFile@27461fd5_129